torch/_inductor/select_algorithm.py

```diff
@@ -1134,6 +1262,17 @@ def generate(  # type: ignore[override]
             "subgraphs": subgraphs,
         }
 
+        cache_result = self._generated_module_cache.get_entry(cache_key, input_nodes)
```

Comment:
Should there be a way to disable the cache?

Reply:
yeh probably let me add that